node.go

```diff
@@ -1,11 +1,80 @@
 package pgs
 
+import (
+	"strings"
+
+	"github.com/golang/protobuf/protoc-gen-go/descriptor"
+)
+
 // Node represents any member of the proto descriptor AST. Typically, the
 // highest level Node is the Package.
 type Node interface {
 	accept(Visitor) error
 }
 
+// NodeMeta contains metadata for a node.
+//
+// See the documentation for .google.protobuf.SourceCodeInfo for more details.
+//
+// https://github.com/google/protobuf/blob/master/src/google/protobuf/descriptor.proto
+type NodeMeta struct {
```

Comment:
since we're not exposing access to NodeMeta let's make it unexported.

Reply:
This was not exported just for demonstration purposes here - all I exposed so far in this PR was the existing functionality. I wanted to add a `NodeMeta() *NodeMeta` function wherever there is a `Comments() string` function (or `NodeMeta() NodeMeta` if you want a copy constructor effectively as well, and changing all the pointer receiver methods to structs on NodeMeta). How does this sound?